src/ars/core/repository.py

```diff
@@ -248,13 +255,21 @@ async def update(  # noqa: C901
             raise missing_iva_error
 
         access_starts = patch_data.access_starts or request.access_starts
```

Comment:
Should we also warn the DS if they try to patch a start date before the current date? Since that data is optional to send, we shouldn't have issues because of having to pass the currently set date.

Reply:
The start date is always "auto-corrected" to be at least the current date.